src/nni_manager/training_service/kubernetes/azureStorageClientUtils.ts

```diff
@@ -69,16 +75,17 @@ export namespace AzureStorageClientUtility {
      * @param fileServerClient
      * @param azureDirectory
      */
-    export async function createDirectoryRecursive(fileServerClient: any, azureDirectory: any, azureShare: any): Promise<void>{
+    export async function createDirectoryRecursive(fileServerClient: any, azureDirectory: string, azureShare: any): Promise<void> {
```

Comment:
I think such client does have certain type. making its type `any` just bypassed the ts-lint, and should not be encouraged.

Reply:
'azureStorage.FileService', I will check the others.

Reply:
I have removed some tslint disable comments just now according to chengmin's comments